CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-https-agent ChangeLog
 
+## 2.2.0 - 2022-TBD
+
+### Added
+- Add additional tests.
```

Comment:
There's some private API surface that gets added that should be mentioned.

```suggestion
- Add additional tests and expose private test helper functions in API.
```

Reply:
Updated 29b8f78